src/main/java/com/purbon/kafka/topology/api/ccloud/CCloudCLI.java

```diff
@@ -30,6 +30,7 @@ public CCloudCLI() {
       return Arrays.stream(items).collect(Collectors.toMap(ServiceAccount::getName, i -> i));
     } catch (IOException | InterruptedException e) {
       handleError(stdout, e);
+      Thread.currentThread().interrupt();
```

Comment:
should this be handled in another catch only for InterruptedExceptions? What are the implications of interrupting when IOException happen?

Reply:
I don't think this is the proper fix for this issue. 

1. The InterruptedException should first of all be handled in the run method where it can be thrown. And then by a re-throw, so the SonarCube rule would not be violated after all.
2. I any way handling it with `Thread.currentThread().interrupt();` is wrong, as the handleError method will throw an IOException so this line will never be reached.